client/shade/pom.xml

```diff
@@ -279,6 +279,13 @@
                                 <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
                                     <manifestEntries>
                                         <Multi-Release>true</Multi-Release>
+                                        <Bundle-ManifestVersion>2</Bundle-ManifestVersion>
+                                        <Bundle-SymbolicName>org.jboss.client</Bundle-SymbolicName>
+                                        <Bundle-Version>1.0</Bundle-Version>
+                                        <Bundle-Name>Test</Bundle-Name>
```

Comment:
The Bundle-Name should be something more descriptive than "Test".

A good candidate could be something derived from client/shade/pom.xml#name, although without "WildFly: "

Reply:
@yersan I made the changes to the bundle name, please review.